### ISSUE TITLE
REFERENCE INTO, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1532]
```abap
TYPES: BEGIN OF ty_tab,
         text TYPE string,
       END OF ty_tab.
DATA lt_message TYPE STANDARD TABLE OF ty_tab.

LOOP AT lt_message REFERENCE INTO DATA(lr_message).
  WRITE lr_message->text.
ENDLOOP.
```

![image](https://user-images.githubusercontent.com/5888506/98523722-84892d80-2276-11eb-8243-4cd2d0ef0e5c.png)
